profiler/wdt_follow_ajax.rst

```diff
@@ -0,0 +1,40 @@
+.. index::
+    single: Profiling: WDT Auto-update after AJAX Request
+
+How to make the Web Debug Toolbar Auto-update after AJAX Requests
+================================================================
+
+For single page applications it would be more convenient if the toolbar 
+showed the information for the most recent AJAX request instead of the 
+initial page load.
+
+By setting the `Symfony-Debug-Toolbar-Replace` header to a value of `1` in the
```

Comment:
The syntax used in Symfony Docs is similar to Markdown, but different in some cases. One of those cases is the use of double backticks for code. This:

```
`Symfony-Debug-Toolbar-Replace`
```

should be written like this:

```
``Symfony-Debug-Toolbar-Replace``
```

Same for the `1` code used in the same phrase.